dev/com.ibm.ws.jsp.2.3_fat/publish/servers/jspJava8Server/server.xml

```diff
@@ -16,6 +16,7 @@
 
     <featureManager>
         <feature>jsp-2.3</feature>
+        <feature>localConnector-1.0</feature>
```

Comment:
Is this necessary?

Reply:
Yes, for the useJDKCompiler test.

Otherwise, we get:
2023-08-29-13:33:21:084 Failed to read a JMXServiceURL from a server's local address file; is the localConnector-1.0 feature enabled? Address File: /Users/volosied/libertyGit/open-liberty/dev/build.image/wlp/usr/servers/jspJava8Server/workarea/com.ibm.ws.jmx.local.address